x-pack/test/detection_engine_api_integration/security_and_spaces/tests/create_index.ts

```diff
@@ -88,7 +91,7 @@ export default ({ getService }: FtrProviderContext) => {
           .expect(403);
         expect(body).to.eql({
           message:
-            'security_exception: action [cluster:admin/ilm/get] is unauthorized for user [t1_analyst], this action is granted by the privileges [read_ilm,manage_ilm,manage,all]',
+            'security_exception: action [cluster:admin/ilm/get] is unauthorized for user [t1_analyst], this action is granted by the cluster privileges [read_ilm,manage_ilm,manage,all]',
```

Comment:
are messages a defined part of the ES API spec? we've been burned a few times by wording changes in these messages causing tests to fail and get skipped. I could imagine a new privilege being added that grants `[cluster:admin/ilm/get]` and the array `[read_ilm,manage_ilm,manage,all]` would then include the new privilege. I don't think that would be considered a breaking change in ES but it would break this test.

Perhaps we can verify part of the message, such as that it begins with `security_exception` instead of asserting the whole string?

Reply:
It shouldn't change often from talking to others about this one when ES was promoted. If it were to change again it could be at either the start, end, or middle of the string. For example they could change the wording of `security_exception:` to something else at the start of it.

From my understanding, during the promotions of elastic search and when people skip tests they don't view the string failures like this one as something to necessarily minimize but rather as a way for Kibana core members and us to double check the wording and the elastic tickets to ensure these changes in the strings are intentful from the elastic team and make sense when it bubbles up to the UI.

Kibana core skips the tests as we're the best to fix them but if we see something such as a misspellings or worse something that looks wrong such as the ending string array of `[read_ilm,manage_ilm,manage,all]` missing in one of the properties, then the diff in the failure is useful for everyone.

We have previously rejected the promotion and asked downstream elastic to issue a fix before the promotion is allowed to move forward in some failure cases rather than do a skip.